<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>user_re</class>
 <widget class="QMainWindow" name="user_re">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>修改用户信息</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/new/prefix1/icon/set.ico</normaloff>:/new/prefix1/icon/set.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="sta_head_jpg">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>头像</string>
    </property>
   </widget>
   <widget class="QLabel" name="user_headjpg">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/new/prefix1/pictrue/base.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="sta_back">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>封面</string>
    </property>
   </widget>
   <widget class="QLabel" name="user_back">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>240</width>
      <height>135</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/new/prefix1/pictrue/base_background.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="re_head">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <underline>true</underline>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
color: rgb(0, 0, 255);</string>
    </property>
    <property name="text">
     <string>修改头像</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="re_back">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>160</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <underline>true</underline>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
color: rgb(0, 0, 255);</string>
    </property>
    <property name="text">
     <string>修改封面</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>用户名</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="user_name">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>260</y>
      <width>240</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>昵称</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edi_nickname">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>310</y>
      <width>240</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="re_nick">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>310</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <underline>true</underline>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
color: rgb(0, 0, 255);</string>
    </property>
    <property name="text">
     <string>修改昵称</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>简介</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="edi_brief">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>360</y>
      <width>241</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="re_brief">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>360</y>
      <width>40</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <underline>true</underline>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
color: rgb(0, 0, 255);</string>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>密码</string>
    </property>
   </widget>
   <widget class="QPushButton" name="re_password">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>500</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <underline>true</underline>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
color: rgb(0, 0, 255);</string>
    </property>
    <property name="text">
     <string>修改密码</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="outlogin">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>550</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>退出登录</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>410</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
